src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+const itemOfList = document.querySelectorAll('li');
+
+function convertingSalary(salary) {
```

Comment:
```suggestion
function convertSalary(salary) {
```
![image](https://user-images.githubusercontent.com/96409563/201897160-484dc101-4154-4eaf-96bf-d7f346fdd1b1.png)